Sources/Utility/Platform.swift

```diff
@@ -18,7 +18,9 @@ public enum Platform {
         case Debian
     }
     
-    public static func currentPlatform() -> Platform? {
+    // Lazily return current platform.
+    public static var currentPlatform = Platform.findCurrentPlatform()
+    private static func findCurrentPlatform() -> Platform? {
```

Comment:
You should be able to use something like:
```
public static var currentPlatform = {
}()
```
which is a standard trick I believe.